setup.cfg

```diff
@@ -3,7 +3,7 @@ exclude = */migrations/*.py,*/manage.py,ve/*
 max-line-length = 88
 
 [tool:pytest]
-python_files=registrations/test*.py registrations/**/test*.py changes/test*.py changes/**/test*.py ndoh_hub/test*.py scripts/**/test*.py eventstore/test*.py eventstore/**/test*.py
+python_files=registrations/test*.py registrations/**/test*.py changes/test*.py changes/**/test*.py ndoh_hub/test*.py scripts/**/test*.py eventstore/test*.py eventstore/**/test*.py ada/test*.py ada/**/test*.py
 addopts = --verbose --ds=ndoh_hub.testsettings --ignore=ve --cov=ndoh_hub --cov=registrations --cov=changes --cov=scripts --cov=eventstore --no-cov-on-fail
```

Comment:
We'll also need to add a `--cov=ada` to have test coverage report on the `ada` folder